src/System.IdentityModel.Tokens/AsymmetricSignatureProvider.cs

```diff
@@ -238,11 +255,15 @@ public override byte[] Sign(byte[] input)
             {
                 throw new ObjectDisposedException(GetType().ToString());
             }
-
+#if DNXCORE50
+            if (rsaCng != null)
+                return rsaCng.SignData(input, hash, RSASignaturePadding.Pkcs1);
```

Comment:
@bartonjs In FxCore RSA.SignData (public byte[] SignData(byte[] data, HashAlgorithmName hashAlgorithm, RSASignaturePadding padding)) has a different signature than RSA in .net 451 public byte[] SignData(byte[] buffer, Object halg). How are we to think about the 'padding'?